libs/domain/feature-loaders/src/snapshot/SnapshotProposalLoader.ts

```diff
@@ -88,56 +108,59 @@ const INFO = {
 @Nested()
 class Space {
     @OptionalProperty()
-    name: string
+    name: string;
     @NonEmptyProperty()
-    id: string
+    id: string;
 }
 
 @Nested()
 class Strategy {
     @NonEmptyProperty()
-    name:string
+    name: string;
 }
 
 @Data({
     info: INFO,
 })
 export class Proposal {
     @NonEmptyProperty()
-    id: string
+    id: string;
     @NonEmptyProperty()
-    author: string
+    author: string;
     @NonEmptyProperty()
-    created: number
+    created: number;
     @OptionalObjectRef(Space)
-    space: Space | undefined
+    space: Space | undefined;
     @OptionalProperty()
-    type:string | undefined
+    type: string | undefined;
     @RequiredArrayRef(Strategy)
-    strategies: Strategy[]
+    strategies: Strategy[];
     @NonEmptyProperty()
-    title:string
+    title: string;
     @OptionalProperty()
-    body: string | undefined
+    body: string | undefined;
     @RequiredStringArrayOf()
-    choices: string[]
+    choices: string[];
     @NonEmptyProperty()
-    start: number
+    start: number;
     @NonEmptyProperty()
-    end: number
+    end: number;
     @NonEmptyProperty()
-    snapshot: string
+    snapshot: string;
     @NonEmptyProperty()
-    state: string
+    state: string;
     @OptionalProperty()
-    link: string | undefined
+    link: string | undefined;
     @OptionalNumberArrayOf()
-    scores: number[] | undefined
+    scores: number[] | undefined;
```

Comment:
I think you haven't answered my question about `type: string | undefined` vs `type?: string`. Or did I miss it?

Reply:
I hadn't gotten to it yet. Done now tho!